From 2a032a5f3777b7c388561104fa94b58231d4619e Mon Sep 17 00:00:00 2001
From: Corey Bryant <corey.bryant@canonical.com>
Date: Fri, 5 Jun 2015 09:41:06 -0400
Subject: [PATCH] Use python-six shim for assertRaisesRegex/p

Python2 uses assertRaisesRegexp while Python3 uses assertRaisesRegex.
Use the compatability shim in six for this:
https://pythonhosted.org/six/#unittest-assertions

Change-Id: I28ce94207567e0b3c28c634119757a1d68e955f2
Closes-Bug: #1462370
---
 keystoneclient/tests/unit/test_session.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/keystoneclient/tests/unit/test_session.py b/keystoneclient/tests/unit/test_session.py
index e0b6fb5..0b69ca4 100644
--- a/keystoneclient/tests/unit/test_session.py
+++ b/keystoneclient/tests/unit/test_session.py
@@ -244,10 +244,11 @@ class SessionTests(utils.TestCase):
         # The exception should contain the URL and details about the SSL error
         msg = _('SSL exception connecting to %(url)s: %(error)s') % {
             'url': self.TEST_URL, 'error': error}
-        self.assertRaisesRegex(exceptions.SSLError,
-                               msg,
-                               session.get,
-                               self.TEST_URL)
+        six.assertRaisesRegex(self,
+                              exceptions.SSLError,
+                              msg,
+                              session.get,
+                              self.TEST_URL)
 
 
 class RedirectTests(utils.TestCase):
-- 
2.1.4

